docs/UserGuide.md

```diff
@@ -13,84 +13,85 @@ can use **RC4HDB** to its fullest potential.
+    * [Editing an existing resident `edit`](#editing-an-existing-resident--edit)
+    * [Deleting an existing resident `delete`](#deleting-a-resident--delete)
+    * [Clearing all residents `clear`](#clearing-all-entries--clear)
+    * [Resident field format](#format-for-resident-fields)
+  * [**Viewing residents**](#viewing-residents)
+    * [Listing all residents `list`](#listing-all-residents--list)
+    * [Showing resident fields `showonly`](#showing-only-some-columns--showonly)
+    * [Hiding resident fields `hideonly`](#hiding-only-some-columns--hideonly)
+    * [Resetting hidden resident fields `reset`](#resetting-hidden-columns--reset)
+    * [Finding residents `find`](#locating-residents-by-name--find)
+    * [Filtering residents `filter`](#filtering-residents-by-field--filter)
+  * [**File management**](#file-management)
+    * [Finding your data](#finding-your-data)
+    * [Editing your data file](#editing-your-data-file)
+    * [Saving your data](#saving-your-data)
+    * [Creating a new data folder `file create`](#creating-a-new-data-folder--file-create)
+    * [Deleting an existing data folder `file delete`](#deleting-an-existing-data-folder--file-delete)
+    * [Switching to a different data folder `file switch`](#switching-to-a-different-data-folder--file-switch)
+    * [Importing resident data from CSV file `file import`](#importing-resident-data-from-csv-file--file-import)
+    * [CSV file format](#csv-file-format)
+  * [**Venue management**](#venue-management)
+    * [Viewing the bookings](#viewing-the-bookings)
+    * [Adding a venue `venue add`](#adding-a-venue--venue-add)
+    * [Deleting a venue `venue delete`](#deleting-a-venue--venue-delete)
+    * [Viewing a venue `venue view`](#viewing-a-venue--venue-view)
+    * [Adding a booking `venue book`](#adding-a-booking-venue-book)
+    * [Deleting a booking `venue unbook`](#deleting-a-booking-venue-unbook)
+    * [Format for venue fields](#format-for-venue-fields)
+  * [**Quality-of-life**](#quality-of-life)
+    * [Keyboard shortcuts](#keyboard-shortcuts)
+    * [Accessing command history](#accessing-command-history)
+  * [**Exiting RC4HDB `exit`**](#exiting-the-program--exit)
+  * [**Upcoming features**](#upcoming-features)
+* [**FAQ**](#faq)
+* [**Glossary**](#glossary-of-terms)
+* [**Authors**](#authors)
+* [**Command Summary**](#command-summary)
+
+---
+
+## **Introduction to RC4HDB**
+
+**RC4HDB** is a desktop application which streamlines the daily workflow of **RC4 housing management staff**, by providing specialised features which solve their resident and venue management needs.
+
+Broadly, **RC4HDB** provides users with the ability to:
+* View and manage resident data
+* View and manage venues and any bookings
+* Keep and manage multiple data files
+* Easily switch over to RC4HDB by importing old data from CSV files
 
 <div markdown="span" class="alert alert-info">
 If you can type fast, RC4HDB can be a convenient and intuitive way to facilitate your day-to-day
```

Comment:
Reframe this to:
RC4HDB is already good as is, but if you're a fast typer, RC4HDB will be even better.
Typing fast is an added bonus, it's not an exclusion

Reply:
Other comments include: insert more hyperlinks which reference the glossary, and to update the images with callouts to make them more effective on the user.